.github/codecov.yml

```diff
@@ -0,0 +1,9 @@
+comment:
+  layout: "header, diff, components, files" # show component info in the PR comment
+
+coverage:
+  status:
+    patch:
+      default:
+        target: 80%
+        informational: true 
```

Comment:
This will enforce patch coverage for the whole repo.

Please create components as done in - https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/master/.github/codecov.yml#L14

and only add patch coverage for the Spanner relevant modules (similar to CODEOWNERS)

Other teams can add their components as they see fit. 

Also add a minimum threshold for the overall coverage of Spanner component.

Reply:
Done.